input_file.rb

```diff
@@ -0,0 +1,61 @@
+require_relative("sales_tax.rb")
+require_relative("print_receipt.rb")
+
+PRODUCTS_ARR = []
+EXEMPT = %w[chocolates book pills chocolate]
+
+class Entry
+  def input
+    no = 0
+    puts "Number of products?"
+    no = gets.chomp.to_i
+    while no > 0
+      data = gets.chomp
+      split_data = data.split
+      qty = split_data[0].to_i
+      price = split_data[-1].to_f
+      p = data.split(" at ")
+      product1 = p[0].delete("/0-9/")
+      prd = product1.strip
+      product = product1.strip
```

Comment:
Why are you storing same value in 2 variables?

Reply:
I am using it because it is giving me product which is imported and not imported, since when i was passing (product) instead of (prd) it was not including (imported) in output because it was manipulating so i created another variable and passed it.
but now it's working.